content/docs/0.8.x/continuous_delivery/multi_stage/index.md

```diff
@@ -5,46 +5,98 @@ weight: 1
 keywords: [0.8.x-cd]
 ---
 
-The definition of a multi-stage delivery manifests in a so-called **shipyard**. It can hold multiple stages with dedicated and opinionated delivery tasks to execute. Following this declarative approach, there is no need to write imperative pipeline code. Keptn takes the shipyard file and creates a task sequence for multi-stage delivery.
+The definition of a multi-stage delivery manifests in a so-called **shipyard**. It can hold multiple stages with dedicated and opinionated delivery tasks to execute. Following this declarative approach, there is no need to write imperative pipeline code. Keptn takes the shipyard and creates a sequence for multi-stage delivery.
 
 ## Declare Shipyard (before creating a project)
 
-The `shipyard.yml` file defines the stages each deployment has to go through until it is released in production. A shipyard file can consist of any number of stages with at least the name property. Besides, each stage can consist of sequence tasks (strategies) from the following list:
+* A shipyard is defined at the level of a project. This means that all services in a project share the same shipyard configuration. 
```

Comment:
I would go with "shipyard definition" instead of "shipyard configuration" but I guess both is fine

Reply:
in the subsequent parts it is always called "definition of ..." thus my input of renaming this

Reply:
valid point - changed it.